xbmc/pvr/PVRManager.cpp

```diff
@@ -594,12 +573,31 @@ bool CPVRManager::SetWakeupCommand(void)
 
 void CPVRManager::OnSleep()
 {
+  SetWakeupCommand();
+  SaveLastPlayedChannel();
```

Comment:
Just cosmetics, but in CPVRManager::Deinit() you are calling these two methods in the the opposite order.